akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -381,8 +430,15 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
 
   val (afrFileChannel, afrFlie, flightRecorder) = initializeFlightRecorder()
 
+  def createFlightRecorderEventSink(): EventSink = {
+    // FIXME there is some concurrency issue with the FlightRecorder, when shutting down.
+    //       It crashes the JVM.
+    // flightRecorder.createEventSink()
+    IgnoreEventSink
```

Comment:
Yes, I changed the shutdown call to be from Future, but that can't really have been safe before either. Synchronized sounds good for this.